src/Reflection/ReflectionClass.php

```diff
@@ -490,6 +497,56 @@ public function hasConstant(string $name) : bool
     }
 
     /**
+     * Get the reflection object of the specified class constant.
+     *
+     * Returns null if not specified.
+     *
+     * @param string $name
+     * @return ReflectionClassConstant|null
+     */
+    public function getReflectionConstant(string $name): ?ReflectionClassConstant
+    {
+        $constants = $this->getReflectionConstants();
+
+        if (!isset($constants[$name])) {
+            return null;
```

Comment:
In original php-reflection this method returns false:
https://github.com/php-src/php/blob/master/ext/reflection/php_reflection.c#L4547